hc/api/management/commands/sendreports.py

```diff
@@ -27,7 +27,7 @@ def add_arguments(self, parser):
             help='Keep running indefinitely in a 300 second wait loop',
         )
 
-    def handle_one_run(self):
+    def handle_month_run(self):
```

Comment:
`handle_month_run`, `handle_week_run`, `and handle_daily_run` could be one method that has arguments for the number of days `1`, `7`, or `30`